lib/events/events.go

```diff
@@ -35,6 +35,7 @@ const (
 	PendingDevicesChanged
 	DevicePaused
 	DeviceResumed
+	RemoteClusterConfigReceived
```

Comment:
@calmh Proposed `RemoteFoldersChanged`. I like that because the "cluster-config" term ins't particularly nice/descriptive, but mirroring the BEP message name has the benefit of making that connection. I am undecided on this.

Reply:
Lots of undecidedness here on my side as well ;-)

IMO if we go back to conveying the actual changed data in this event **instead of** FolderCompletion, it would make sense to rename it.  The separate endpoint for retrieving the initial state feels too much for me though.

If it's named `RemoteFoldersChanged`, we should define what exactly has changed.  There is a lot of information in that message, where any bit could be considered a change.  Right now we're only interested in the presence and the `paused` flag.  Anything else should at least not trigger a new FolderSummary.

Reply:
I am fine leaving it as is. Except one more thing regarding the naming of the status quo, at least here I am right now less undecided: I don't think there's any need for the "remote" in the name, any received cluster config comes from a remote by definition.